KeeAgent/ExtensionMethods.cs

```diff
@@ -394,13 +405,13 @@ public static ISshKey GetSshKey(this PwEntry entry)
         parameter = (AsymmetricKeyParameter)dialog.Result;
       }
       else {
-        parameter = privateKey.Decrypt(() => entry.GetPassphrase());
+        parameter = privateKey.Decrypt(getPassphrase, null, newComment => comment = newComment);
       }
 
       var key = new SshKey(
         publicKey.Parameter,
         parameter,
-        publicKey.Comment,
+        comment,

```

Comment:
One thing I would like to do going forward is to make a clear distinction between the public key comment and the private key comment. 

Here, we want to make sure that we prefer the public key comment since we could have multiple public keys with certificates that use the same private key. We would want to see the comment from the public key so that we can tell the different entries in the agent apart. This should only fall back to the private key comment if no public key comment was given.

Reply:
OK, I'll change it.